shardingsphere-jdbc/shardingsphere-jdbc-core/src/test/java/org/apache/shardingsphere/driver/jdbc/core/statement/EncryptPreparedStatementTest.java

```diff
@@ -84,7 +84,7 @@ public void assertInsertWithBatchExecute() throws SQLException {
     
     @Test
     public void assertInsertWithExecuteWithGeneratedKey() throws SQLException {
-        try (PreparedStatement statement = getEncryptConnection().prepareStatement(INSERT_GENERATED_KEY_SQL, Statement.RETURN_GENERATED_KEYS)) {
```

Comment:
@zjcnb Why modify these two unit tests?

Reply:
When I ran the unit tests, these two didn't passed, but now they pssed, so I'll restore them